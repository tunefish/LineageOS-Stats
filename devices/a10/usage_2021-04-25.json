{
  "10.3": 1,
  "16.0": 75,
  "17.1": 951,
  "18.1": 548,
  "none": 506
}