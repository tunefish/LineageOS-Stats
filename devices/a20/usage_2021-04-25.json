{
  "17.1": 2435,
  "18.1": 273,
  "none": 28
}