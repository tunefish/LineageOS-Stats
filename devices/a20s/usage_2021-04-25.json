{
  "15.1": 1,
  "17.1": 2497,
  "none": 1
}