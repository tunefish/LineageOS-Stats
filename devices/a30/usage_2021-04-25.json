{
  "17.1": 38,
  "18.0": 776,
  "18.1": 497,
  "none": 4
}