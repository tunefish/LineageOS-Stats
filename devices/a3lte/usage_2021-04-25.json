{
  "15.1": 761,
  "16.0": 282,
  "17.1": 963
}