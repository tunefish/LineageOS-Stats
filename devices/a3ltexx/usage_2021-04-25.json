{
  "14.1": 1217
}