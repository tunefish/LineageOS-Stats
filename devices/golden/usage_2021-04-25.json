{
  "13.0": 1344,
  "14.1": 4724,
  "15.1": 6,
  "17.1": 12,
  "none": 1
}